README.md

```diff
@@ -11,12 +11,24 @@ http://harvesthq.github.io/chosen/
 For **downloads**, see:
 https://github.com/harvesthq/chosen/releases/
 
-### Bower Installation
+### Package managers
 
-Chosen is available for Bower installation:
-`bower install chosen`
+Chosen is available through [Bower](https://bower.io/) and [npm](https://www.npmjs.com),
+_however, the package names are not the same_
 
-The compiled files for the Bower package are automatically generated and stored in a [2nd Chosen repository](https://github.com/harvesthq/bower-chosen). No pull requests will be accepted to that repository.
+To install with Bower:
+
+```
+bower install chosen
+```
+
+To install with npm:
+
+```
+npm install chosen-js
+```
+
+The compiled files for these packages are automatically generated and stored in a [2nd Chosen repository](https://github.com/harvesthq/chosen-package). No pull requests will be accepted to that repository.
```

Comment:
No it doesn't, but the archive being uploaded is based on the contents of that repo by using the deploy option provided by Travis.